tests/testthat/math_token_spacing/strict_math_spacing_zero_all_but_power-out.R

```diff
@@ -0,0 +1 @@
+1++1-3/23*3 ^ 4
```

Comment:
I wonder if the other way round would be more usual, because of operator precedence:

```r
+1 + +1 - 3 / 23 * 3^4
```

Reply:
Right, I can adapt the test. The default should remain as in the tidyverse style guide, so all operators have spacing around them I think.